resources/views/includes/head.blade.php

```diff
@@ -17,4 +17,6 @@
   @include('feed::links')
 @endif
 
+<script src="dist/Larapass.js"></script>
```

Comment:
Why isn't this compiled in with the rest of the front end?

Reply:
It doesn't work if it is compiled. :(